src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import lotto.enums.LottoRank;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoResult {
+    Map<LottoRank, Integer> winningCount = new HashMap<>();
+    double profitRate;
```

Comment:
만약 클래스 변수로 가져가실 계획이시라면 접근제한자가 추가 되어야 할 것 같습니다 😄 
```suggestion
    private double profitRate;
```

두 가지 의견을 드리고 싶은데요 
하나는 굳이 클래스 변수로 가져가지 말고 메소드 호출 시 바로 결과 값을 반환 하도록만 해도 충분할 것 같습니다 
클래스 변수로 할 경우 클래스 내에서 모든 `method`들이 접근 가능하기 때문에 실수를 유발하거나 다른 `method`에서 값을 변경할 수 있는 통로가 열려있는 것과 같습니다 😰 

그리고 나머지는 이미 구매금액을 담당하고 있는 `Money`라는 객체를 만들어 주셨습니다 😄 
그 객체에게 당첨금액을 주고 수익률을 "**구해달라**" 고 해보시는 건 어떨까요?? 🤔 

이 부분도 한번 개선검토 해주시면 감사하겠습니다 😄

Reply:
수익률을 구하는 역할을 Money에 위임할 생각은 못하였는데 좋은 역할 분담이 될 것 같아 반영하였습니다.
의견 감사드립니다 ^^!